rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -276,16 +277,38 @@ def update_kl(self, sampled_kl):
         else:
             return self.kl_coeff_val
 
-        # Update the tf Variable (via session call for tf).
+        # Make sure, new value is also stored in graph/tf variable.
+        self._set_kl_coeff(self.kl_coeff_val)
+
+        # Return the current KL value.
+        return self.kl_coeff_val
+
+    def _set_kl_coeff(self, new_kl_coeff):
+        # Set the (off graph) value.
+        self.kl_coeff_val = new_kl_coeff
+
+        # Update the tf/tf2 Variable (via session call for tf or `assign`).
         if self.framework == "tf":
             self.get_session().run(
                 self._kl_coeff_update,
                 feed_dict={self._kl_coeff_placeholder: self.kl_coeff_val},
             )
         else:
             self.kl_coeff.assign(self.kl_coeff_val, read_value=False)
-        # Return the current KL value.
-        return self.kl_coeff_val
+
+    @override(Policy)
+    def get_state(self) -> Union[Dict[str, TensorType], List[TensorType]]:
```

Comment:
This is the part I never understand about Mixins, what if LearningRateSchedule or some other Mixins also need to update get/set_state()?
Will this still work?

Reply:
This would fail, b/c `super()` would only resolve to one of the super classes (I think first one)! That's why mixins are bad :D

Reply:
Can we add a comment above this function about this problem please?
Is it too late to add a proposal for "deprecating Mixins"?? :)